features/install-cli.feature

```diff
@@ -52,6 +54,10 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary "docker" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli docker --cli-version 1.12.1`
```

Comment:
Any reason we only test the _docker_ download? Where are tests for downloading _oc_?